go/exocom-mock/bin/spec.cmd

```diff
@@ -1,2 +0,0 @@
-go install
```

Comment:
We only have a file like it on `exocom-mock` and its not up to date. I didn't think its worth keeping.